### PR TITLE
Awards logic

### PR DESCRIPTION
**Pull request by freedannyv**
# Type of PR
New Feature
#42 
Put an `x` in the brackets for the type(s) of PR this is. You may tick more than one.

[ ] Bug Fix
[ ] Refactor
[x] New Feature
[ ] Update to Existing Feature
[ ] Other (state below)

# Description

**For New Features:** What feature are we adding? Explain your technical approach to doing this.
Awards Ceremony screen at the end of the game. Logic is in but needs further testing. We loop through 5 awards at the end of the game. I also updated the design to show a description of the award, and output the result of the player (eg. their score if they won Player of the Day). I used Photoshop to edit the ribbons so that they are the same size etc, and updated them in the file.

I have also implemented the par to show on the screens for each hole. I would like to refactor this into Vuex, however.

Provide steps on how one can test your changes here. e.g.
You can go through the course adding results and when you get to the award screen, you'll be delighted with a little awards ceremony

### CODE REVIEW

**Review by paulchrisluke — CHANGES_REQUESTED** [review]
button styling
https://deploy-preview-49--weareminigolf.netlify.app/game-scores
image

is this supposed to be working?


image

Clicking outside rule box should close it

File: src/views/Awards.vue [hunk 1]
```diff
@@ -55,35 +35,207 @@
...
+
+      playersInfo: [],
+      coursePar: []
     };
   },
   created() {
     this.getGameDetails()
       .then(() => {
+        this.coursePar = this.getPar;
+        console.log(this.coursePar);
```
Comment (giteshk11) [inline src/views/Awards.vue:64]: remove any console.logs

File: src/views/NewHole.vue [hunk 2]
```diff
@@ -53,10 +58,16 @@ export default {
       });
       console.log(this.courseGrid.squareInfo);
```
Comment (giteshk11) [inline src/views/NewHole.vue:59]: here also

**Review by giteshk11 — CHANGES_REQUESTED** [review]

File: src/views/Awards.vue (hunk 1 above)
Comment (freedannyv) [reply src/views/Awards.vue:64]: Gotcha

**Comment by freedannyv** [comment]
@giteshk11 updates made, please have a look. Thanks!

**Pull request merged**